app/models/user.rb

```diff
@@ -6,4 +6,5 @@ class User < ApplicationRecord
   def recent_post
     posts.order('created_at Desc').limit(3)
   end
+
```

Comment:
- [ ] The `User.rb`  association should have the foreign keys with ID.

Reply:
Done :white_check_mark: